--- CMakeLists.txt.orig	2016-10-02 03:01:24.000000000 +0300
+++ CMakeLists.txt	2016-10-15 15:43:41.000000000 +0300
@@ -55,7 +55,6 @@
 endif()
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR})
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/clew/include)
 # include_directories("${CMAKE_INSTALL_PREFIX}/include")
 
 include_directories(thirdparty/lua-5.1.5/src)
@@ -65,8 +64,7 @@
 SET(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 if(USE_CLEW)
-    INCLUDE("${CMAKE_MODULE_PATH}/build_clew.cmake")
-    INCLUDE_DIRECTORIES(${clew_INCLUDE_DIRS})
+    find_library(CLEW_LIBRARIES NAMES clew libclew.a)
 endif()
 
 if(PROVIDE_LUA_ENGINE)
