--- CMakeLists.txt.orig	2015-11-30 01:26:19.000000000 +0300
+++ CMakeLists.txt	2015-12-27 22:39:54.000000000 +0300
@@ -50,7 +50,6 @@
 endif()
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR})
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/clew/include)
 # include_directories("${CMAKE_INSTALL_PREFIX}/include")
 
 include_directories(thirdparty/lua-5.1.5/src)
@@ -60,8 +59,7 @@
 SET(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 if(USE_CLEW)
-    INCLUDE("${CMAKE_MODULE_PATH}/build_clew.cmake")
-    INCLUDE_DIRECTORIES(${CLEW_INCLUDE_DIRS})
+    find_library(CLEW_LIBRARIES NAMES clew libclew.a)
 endif()
 
 if(PROVIDE_LUA_ENGINE)
