diff -dPNur library.orig/blas/AutoGemm/KernelOpenCL.py library/blas/AutoGemm/KernelOpenCL.py
--- library.orig/blas/AutoGemm/KernelOpenCL.py	2016-01-14 21:36:50.000000000 +0300
+++ library/blas/AutoGemm/KernelOpenCL.py	2016-01-14 22:16:33.000000000 +0300
@@ -30,7 +30,7 @@
   prec = kernel.getName()[0].lower()
   if prec == "d" or prec == "z":
     kStr += endLine
-    kStr += "#pragma OPENCL EXTENSION cl_khr_fp64 : enable" + endLine
+    kStr += "_Pragma(\\\"OPENCL EXTENSION cl_khr_fp64 : enable\\\") " + endLine
 
   ####################################
   # kernel parameters
diff -dPNur library.orig/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NN_B0_MX048_NX048_KX08_src.cpp library/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NN_B0_MX048_NX048_KX08_src.cpp
--- library.orig/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NN_B0_MX048_NX048_KX08_src.cpp	2016-01-14 21:36:50.000000000 +0300
+++ library/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NN_B0_MX048_NX048_KX08_src.cpp	2016-01-14 22:04:34.000000000 +0300
@@ -18,7 +18,7 @@
 const unsigned int dgemm_Col_NN_B0_MX048_NX048_KX08_unroll = 8;
 
 const char * const dgemm_Col_NN_B0_MX048_NX048_KX08_src = STRINGIFY(
-#pragma OPENCL EXTENSION cl_khr_fp64 : enable \n
+_Pragma("OPENCL EXTENSION cl_khr_fp64 : enable") \n
 
 #define  M6x6 \
             rA[0] = lA[offA + 0];\
diff -dPNur library.orig/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NN_B1_MX048_NX048_KX08_src.cpp library/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NN_B1_MX048_NX048_KX08_src.cpp
--- library.orig/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NN_B1_MX048_NX048_KX08_src.cpp	2016-01-14 21:36:50.000000000 +0300
+++ library/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NN_B1_MX048_NX048_KX08_src.cpp	2016-01-14 22:04:40.000000000 +0300
@@ -18,7 +18,7 @@
 const unsigned int dgemm_Col_NN_B1_MX048_NX048_KX08_unroll = 8;
 
 const char * const dgemm_Col_NN_B1_MX048_NX048_KX08_src = STRINGIFY(
-#pragma OPENCL EXTENSION cl_khr_fp64 : enable   \n
+_Pragma("OPENCL EXTENSION cl_khr_fp64 : enable") \n
 
 #define  M6x6 \
             rA[0] = lA[offA + 0];                       \
diff -dPNur library.orig/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NT_B0_MX048_NX048_KX08_src.cpp library/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NT_B0_MX048_NX048_KX08_src.cpp
--- library.orig/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NT_B0_MX048_NX048_KX08_src.cpp	2016-01-14 21:36:50.000000000 +0300
+++ library/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NT_B0_MX048_NX048_KX08_src.cpp	2016-01-14 22:04:44.000000000 +0300
@@ -18,7 +18,7 @@
 const unsigned int dgemm_Col_NT_B0_MX048_NX048_KX08_unroll = 8;
 
 const char * const dgemm_Col_NT_B0_MX048_NX048_KX08_src = STRINGIFY(
-#pragma OPENCL EXTENSION cl_khr_fp64 : enable   \n
+_Pragma("OPENCL EXTENSION cl_khr_fp64 : enable") \n
 \n
 \ntypedef union _GPtr {
 \n  __global float *f;
diff -dPNur library.orig/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NT_B1_MX048_NX048_KX08_src.cpp library/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NT_B1_MX048_NX048_KX08_src.cpp
--- library.orig/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NT_B1_MX048_NX048_KX08_src.cpp	2016-01-14 21:36:50.000000000 +0300
+++ library/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_NT_B1_MX048_NX048_KX08_src.cpp	2016-01-14 22:04:49.000000000 +0300
@@ -18,7 +18,7 @@
 const unsigned int dgemm_Col_NT_B1_MX048_NX048_KX08_unroll = 8;
 
 const char * const dgemm_Col_NT_B1_MX048_NX048_KX08_src = STRINGIFY(
-#pragma OPENCL EXTENSION cl_khr_fp64 : enable   \n
+_Pragma("OPENCL EXTENSION cl_khr_fp64 : enable") \n
 \n
 \ntypedef union _GPtr {
 \n  __global float *f;
diff -dPNur library.orig/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_TN_B0_MX048_NX048_KX08_src.cpp library/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_TN_B0_MX048_NX048_KX08_src.cpp
--- library.orig/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_TN_B0_MX048_NX048_KX08_src.cpp	2016-01-14 21:36:50.000000000 +0300
+++ library/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_TN_B0_MX048_NX048_KX08_src.cpp	2016-01-14 22:05:01.000000000 +0300
@@ -18,7 +18,7 @@
 const unsigned int dgemm_Col_TN_B0_MX048_NX048_KX08_unroll = 8;
 
 const char * const dgemm_Col_TN_B0_MX048_NX048_KX08_src = STRINGIFY(
-#pragma OPENCL EXTENSION cl_khr_fp64 : enable   \n
+_Pragma("OPENCL EXTENSION cl_khr_fp64 : enable") \n
 
 __attribute__( (reqd_work_group_size(8, 8, 1)) )
 __kernel void dgemm_Col_TN_B0_MX048_NX048_KX08_src (
diff -dPNur library.orig/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_TN_B1_MX048_NX048_KX08_src.cpp library/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_TN_B1_MX048_NX048_KX08_src.cpp
--- library.orig/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_TN_B1_MX048_NX048_KX08_src.cpp	2016-01-14 21:36:50.000000000 +0300
+++ library/blas/AutoGemm/UserGemmKernelSources/dgemm_Col_TN_B1_MX048_NX048_KX08_src.cpp	2016-01-14 22:05:06.000000000 +0300
@@ -18,7 +18,7 @@
 const unsigned int dgemm_Col_TN_B1_MX048_NX048_KX08_unroll = 8;
 
 const char * const dgemm_Col_TN_B1_MX048_NX048_KX08_src = STRINGIFY(
-#pragma OPENCL EXTENSION cl_khr_fp64 : enable   \n
+_Pragma("OPENCL EXTENSION cl_khr_fp64 : enable") \n
 
 __attribute__( (reqd_work_group_size(8, 8, 1)) )
 __kernel void dgemm_Col_TN_B1_MX048_NX048_KX08_src (
diff -dPNur library.orig/blas/trtri/diag_dtrtri_lower_128_16.cpp library/blas/trtri/diag_dtrtri_lower_128_16.cpp
--- library.orig/blas/trtri/diag_dtrtri_lower_128_16.cpp	2016-01-14 21:36:50.000000000 +0300
+++ library/blas/trtri/diag_dtrtri_lower_128_16.cpp	2016-01-14 22:04:06.000000000 +0300
@@ -23,9 +23,9 @@
 #define ONE               ( 1.0) \n
 #ifdef DOUBLE_PRECISION \n
 #ifdef cl_khr_fp64 \n
-#pragma OPENCL EXTENSION cl_khr_fp64 : enable \n
+_Pragma("OPENCL EXTENSION cl_khr_fp64 : enable") \n
 #else \n
-#pragma OPENCL EXTENSION cl_amd_fp64 : enable \n
+_Pragma("OPENCL EXTENSION cl_amd_fp64 : enable") \n
 #endif \n
 #endif \n
 __kernel void diag_dtrtri_lower_128_16_src(\n
diff -dPNur library.orig/blas/trtri/diag_dtrtri_upper_128_16.cpp library/blas/trtri/diag_dtrtri_upper_128_16.cpp
--- library.orig/blas/trtri/diag_dtrtri_upper_128_16.cpp	2016-01-14 21:36:50.000000000 +0300
+++ library/blas/trtri/diag_dtrtri_upper_128_16.cpp	2016-01-14 22:04:12.000000000 +0300
@@ -21,9 +21,9 @@
 #define ONE               ( 1.0) \n
 #ifdef DOUBLE_PRECISION \n
 #ifdef cl_khr_fp64 \n
-#pragma OPENCL EXTENSION cl_khr_fp64 : enable \n
+_Pragma("OPENCL EXTENSION cl_khr_fp64 : enable") \n
 #else \n
-#pragma OPENCL EXTENSION cl_amd_fp64 : enable \n
+_Pragma("OPENCL EXTENSION cl_amd_fp64 : enable") \n
 #endif \n
 #endif \n
 __kernel void diag_dtrtri_upper_128_16_src(\n
diff -dPNur library.orig/blas/trtri/diag_dtrtri_upper_192_12.cpp library/blas/trtri/diag_dtrtri_upper_192_12.cpp
--- library.orig/blas/trtri/diag_dtrtri_upper_192_12.cpp	2016-01-14 21:36:50.000000000 +0300
+++ library/blas/trtri/diag_dtrtri_upper_192_12.cpp	2016-01-14 22:04:24.000000000 +0300
@@ -21,9 +21,9 @@
 #define ONE               ( 1.0) \n
 #ifdef DOUBLE_PRECISION \n
 #ifdef cl_khr_fp64 \n
-#pragma OPENCL EXTENSION cl_khr_fp64 : enable \n
+_Pragma("OPENCL EXTENSION cl_khr_fp64 : enable") \n
 #else \n
-#pragma OPENCL EXTENSION cl_amd_fp64 : enable \n
+_Pragma("OPENCL EXTENSION cl_amd_fp64 : enable") \n
 #endif \n
 #endif \n
 __kernel void diag_dtrtri_upper_192_12_src(\n
