--- library/blas/AutoGemm/AutoGemmTools/AutoGemmPreCompileKernels.cpp.orig	2016-01-14 20:25:35.000000000 +0300
+++ library/blas/AutoGemm/AutoGemmTools/AutoGemmPreCompileKernels.cpp	2016-01-14 20:25:47.000000000 +0300
@@ -40,7 +40,7 @@
 #define stat _stat
 #endif
 
-#include "CL/opencl.h"
+#include "OpenCL/opencl.h"
 //#include "naive_blas.cpp"
 //using namespace NaiveBlas;
 #include "AutoGemmIncludes/AutoGemmKernelsToPreCompile.h"
