--- clBLASConfig.cmake.in.orig	2015-11-30 11:13:30.000000000 +0300
+++ clBLASConfig.cmake.in	2015-11-30 11:13:58.000000000 +0300
@@ -1,3 +1,3 @@
 include(${CMAKE_CURRENT_LIST_DIR}/clBLASTargets.cmake)
-get_filename_component(CLBLAS_INCLUDE_DIRS ${CMAKE_CURRENT_LIST_DIR}/@reldir@/include ABSOLUTE)
+get_filename_component(CLBLAS_INCLUDE_DIRS %%PREFIX%%/include ABSOLUTE)
 set(CLBLAS_LIBRARIES clBLAS)
