--- setup.py.orig	2016-01-12 01:36:10.000000000 +0300
+++ setup.py	2016-01-12 01:38:40.000000000 +0300
@@ -58,8 +58,6 @@
 libraries.append('DeepCL')
 
 library_dirs = []
-library_dirs.append('../dist/lib')
-library_dirs.append('../dist/lib/import')
 
 if osfamily == 'Linux':
     runtime_library_dirs = ['.']
@@ -73,7 +71,7 @@
 ext_modules = [
     Extension("PyDeepCL",
               sources=sources,
-              include_dirs=['mysrc', 'mysrc/lua', '/usr/local/include/deepcl', '/usr/local/include/easycl'],
+              include_dirs=['@prefix@/include/deepcl', '@prefix@/include/easycl'],
               library_dirs=library_dirs,
               libraries=libraries,
               extra_compile_args=compile_options,
