--- CMakeLists.txt.orig	2016-03-23 15:31:08.000000000 +0300
+++ CMakeLists.txt	2016-03-26 23:31:18.000000000 +0300
@@ -71,8 +71,6 @@
 INCLUDE_DIRECTORIES(${CLBLAS_INCLUDE_DIRS})
 ## TARGET_link_LIBRARIES(clBLAS ${EASYCL_LIBRARIES})
 
-ADD_DEPENDENCIES(clBLAS-external EasyCL)
-
 include_directories(.)
 include_directories(src)
 include_directories(qlearning)
@@ -231,10 +229,6 @@
 endif()
 include_directories(thirdparty/gtest)
 
-add_executable(deepcl_unittests ${UNITTEST_SOURCES} test/gtest_main.cpp test/GtestGlobals.cpp
-    test/TestArgsParser.cpp)
-target_link_libraries(deepcl_unittests deepcl_gtest)
-
 if(PROTOTYPING)
     add_executable(clconvolve-fixedweights prototyping/clconvolve-fixedweights.cpp src/util/stringhelper.cpp)
     add_executable(learn_scenario_image prototyping/qlearning/learnScenarioImage.cpp 
@@ -243,15 +237,6 @@
     target_link_libraries(learn_scenario_image DeepCL)
 endif(PROTOTYPING)
 
-target_link_libraries(deepcl_unittests DeepCL)
-
-if(LIBPNGPP_AVAILABLE)
-    target_link_libraries(deepcl_unittests ${PNG_LIBRARY})
-endif(LIBPNGPP_AVAILABLE)
-
-add_executable(testgtestsupp test/testgtestsupp.cpp thirdparty/gtest/gtest_main.cc)
-target_link_libraries(testgtestsupp deepcl_gtest)
-
 if(BUILD_PYTHON_WRAPPERS)
     add_subdirectory(python)
 #    add_dependencies(PyDeepCL DeepCL)
@@ -259,10 +244,8 @@
 endif()
 
 INSTALL(DIRECTORY src/ DESTINATION include/deepcl FILES_MATCHING PATTERN *.h)
-INSTALL(PROGRAMS src/activate.sh DESTINATION bin)
-INSTALL(PROGRAMS src/activate.bat DESTINATION bin)
 #INSTALL(DIRECTORY EasyCL/ DESTINATION include/easycl FILES_MATCHING PATTERN *.h)
-INSTALL(TARGETS DeepCL deepcl_train deepcl_predict deepcl_unittests deepcl_gtest
+INSTALL(TARGETS DeepCL deepcl_train deepcl_predict
     EXPORT DeepCLTargets
     RUNTIME DESTINATION bin
     ARCHIVE DESTINATION lib
@@ -276,7 +259,7 @@
 if(WIN32 AND NOT CYGWIN)
   set(DEF_INSTALL_CMAKE_DIR CMake)
 else()
-  set(DEF_INSTALL_CMAKE_DIR lib/cmake/DeepCL)
+  set(DEF_INSTALL_CMAKE_DIR @cmake_share_module_dir@/DeepCL)
 endif()
 set(INSTALL_CMAKE_DIR ${DEF_INSTALL_CMAKE_DIR})
  
