--- setup.py.orig	2016-12-04 22:26:45.000000000 +0300
+++ setup.py	2016-12-06 15:10:16.000000000 +0300
@@ -35,8 +35,6 @@
       packages=find_packages(),
       # this will use MANIFEST.in during install where we specify additional files,
       # this is the golden line
-      include_package_data=True,
-      data_files=[('xgboost', LIB_PATH)],
       license='Apache-2.0',
       classifiers=['License :: OSI Approved :: Apache Software License'],
       url='https://github.com/dmlc/xgboost')
