--- xgboost/libpath.py.orig	2016-12-03 17:08:11.000000000 +0300
+++ xgboost/libpath.py	2016-12-03 17:08:00.000000000 +0300
@@ -21,9 +21,7 @@
     """
     curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
     # make pythonpack hack: copy this directory one level upper for setup.py
-    dll_path = [curr_path, os.path.join(curr_path, '../../lib/'),
-                os.path.join(curr_path, './lib/'),
-                os.path.join(sys.prefix, 'xgboost')]
+    dll_path = ['@prefix@/lib']
     if os.name == 'nt':
         if platform.architecture()[0] == '64bit':
             dll_path.append(os.path.join(curr_path, '../../windows/x64/Release/'))
